src/integrationtest/python/GeoAttribute_tests.py

```diff
@@ -0,0 +1,48 @@
+# Name: Debasish Kanhar
+
+import unittest
+from subprocess import Popen, PIPE
+from JanuaGraphContainer import JanusGraphContainer
```

Comment:
JanuaGraphContainer is misspelled in all 3 files included in this PR, please fix all occurrences since I can't figure out how to add the in-line comment on `src/integrationtest/python/TextAttribute_tests.py`.